source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -257,8 +203,8 @@ Use your cluster
 
```

Comment:
I don't this this is right anymore. I installed Clear Linux on 12/5 using that version and the default plugin_dir was "/opt/cni/bin"

Reply:
Also, I think the command from Manohar would work for every installation but I have not tested.  I don't think linking just the loopback would be adequate or rather if you use the command below I believe it will allow plugin_dir to be /opt/cni/bin and flannel and weave would just work.  I think this entire section could be collaped just to the below command.  You might want to put some text along the lines of ... Clear Linux installs CNI plugins that are part of the cloud-native-basic bundle to /usr/libexec/cni because that is a directory that Clear Linux's installer swupd can manage to ensure users can use swupd's verify feature to repair a system back to a known good state. Some CNI plugins like Weave auto install themselves to /opt/cni/bin.  To make sure that all of the common CNI plugins work smoothly on Clear Linux symbolically link the two directories."  something like that

sudo ln -s /usr/libexec/cni /opt/cni/bin

I have not tested this command on anything other than flannel.

Reply:
OK I need Mikko or someone else to verify. That command did not work for me when trying to create a weave network and the existing command "sudo ln -s /usr/libexec/cni/loopback /opt/cni/bin/loopback" also did not work.

kubeadm v1.12.3   crio version v1.12.3

From journalctl
Error adding network: failed to find plugin "loopback" in path [/opt/cni/bin/ /opt/loopback/bin]
Error while adding to cni lo network: failed to find plugin "loopback" in path [/opt/cni/bin/ /opt/loopback/bin]
Error deleting network: failed to find plugin "portmap" in path [/opt/cni/bin/ /opt/weave-net/bin]

Reply:
@mythi, @eadamsintel, please see latest commit and provide comments. I replaced/collapsed  the above command, and I took inspiration from @eadamsintel to include a `note` below the newly added command.